README.md

```diff
@@ -111,12 +114,14 @@ The pipeline can be run using any of the following `make` commands:
 ## Credits and contacts
 
 - Research Luxembourg COVID-19 Taskforce WP6, in the person of Alexander Skupin: supervision and coordination. Contact: alexander.skupin@uni.lu.
-- Daniele Proverbio: R_eff code development, website ideation and content creation
-- [Atte if we include his code]
-- [credits to R3 team for website development and deployment]
-- [UniLu/LCSB: website hosting and maintenance?]
+- [Daniele Proverbio](https://github.com/daniele-proverbio): R_eff code development, website ideation and content creation
+- [Atte Aalto](https://github.com/AtteAalto): R_t code development
+- [Laurent Heirendt](https://github.com/laurentheirendt), [Jacek Leboida](https://github.com/jLebioda), [Christophe Trefois](https://github.com/trefex) and the LCSB R3 team: docker and website development and deployment
+
+   <img src="docs/logos.png" alt="logos" width="350"/>  
```

Comment:
can you leave the logos as separate files? You can see an example here: https://github.com/LCSB-BioCore/COBREXA.jl/blob/master/README.md

Reply:
I tried first, but they were aligned badly. What benefit would it bring?

Reply:
The benefit to separate the logos is that they can be used somewhere else or replaced easily 😃